backend/src/schema/resolvers/follow.spec.js

```diff
@@ -129,6 +129,16 @@ describe('follow', () => {
         data: { followUser: expectedUser },
         errors: undefined,
       })
+
+      // Test to make sure FOLLOWS relationship has "createdAt" date.
+      const relation = await neode.cypher(
+        'MATCH (user:User {id: {id}})-[relationship:FOLLOWS]->(followed:User) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+        { id: 'u1' },
+      )
+      const relationshipProperties = relation.records.map(
+        record => record.get('relationship').properties.createdAt,
+      )
+      expect(relationshipProperties[0]).toEqual(expect.any(String))
```

Comment:
Usually you would try to have one assertion per spec. You can easily create a new spec by saying:
```js
it('adds `createdAt` to `FOLLOW` relationship', async () => { ... })
```

Redundancy is not a problem with software tests.

Reply:
@roschaefer I like your suggestion - "Redundancy is not a problem with software tests." I was thinking in terms of code.
Much appreciated your review. Implemented changes as you requested.